tools/downloader/README.md

```diff
@@ -59,6 +59,13 @@ option:
 The `--all` option can be replaced with other filter options to download only
 a subset of models. See the "Shared options" section.
 
+You may use `--precisions` flag to specify comma separated precisions of weights
+to be downloaded.
+
+```sh
+./downloader.py --name face-detection-retail-0004 --precision FP16,INT8
```

Comment:
@dkurt `--precisions`, otherwise it's not working

Reply:
@grib0ed0v, nice catch, thank you!